nixos/modules/services/web-apps/nextcloud.nix

```
@@ -1058,7 +1058,7 @@ in {
             '') [ "nix-apps" "apps" ]}
 
             # Do not install if already installed
-            if [[ ! -e ${datadir}/config/config.php ]]; then
+            if [[ ! -e ${datadir}/config/config.php ]] || [[ ! -s ${datadir}/config/config.php ]]; then
```

```suggestion
            if [[ ! -s ${datadir}/config/config.php ]]; then
```

I think `-s` implies `-e`.